nilearn/regions/region_extractor.py

```diff
@@ -415,27 +458,41 @@ def __init__(
         smoothing_fwhm=6,
         standardize=False,
         standardize_confounds=True,
+        high_variance_confounds=False,
```

Comment:
changes in API but this parameter exists in the parent class
so to me this is a bug fix more than an enhancement because users could have done